### PR TITLE
Plugin page updates

### PR DESCRIPTION
**Pull request by jrastaban**
## Summary

**Updates the WordPress plugin page to include Solr and improve the look with some new assets.

## Effect

The following changes are already committed:

- Added the Apache Solr plugin to the list
- Added banners for each plugin

## Remaining Work

The following changes still need to be completed:

- [x] Should the migration plugin be added? (https://wordpress.org/plugins/bv-pantheon-migration/) - No
- [ ] ~Do we like how the banners look?~
- [ ] ~Optimize the banners.~

86'd the banners -Alexf

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by alexfornuto** [comment]
Re Migration Plugin: I'd say no, because we use it to migrate sites but it's not something customers will want to install on their Pantheon-hosted site, once they've arrived.

**Comment by sugaroverflow** [comment]
I agree with that, thanks @alexfornuto 😄  That plugin is already mentioned on the [Migrate Sites to Pantheon doc](https://pantheon.io/docs/migrate) so I think we're good.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5658 | Difference |
| --- | --- | --- | --- |
| Performance Score | `38` | `24` | `-14` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5658` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/20237/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/20237/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by alexfornuto** [comment]
I _sort of_ like the banners, but I dislike how two plugins in the list don't have them, making them stand out.

I defer to @carolyn-shannon for style choices like this.

**Comment by jrastaban** [comment]
All the plugins have them. Those two sections are subsets of the MU plugin. I was hoping the banner would make it more apparent because I thought the same thing when I pulled up the page. maybe we could style it slightly different to make that clear?

**Comment by jrastaban** [comment]
I doubt they were optimized, do you have a specific tool you recommend? I've used ImageOptim in the past. 

Design also made some 722x250 banners. I see Gatsby is automatically adding adjustments for screen widths but I'm not familiar with how to specify what image to use at different viewport widths.

**Comment by alexfornuto** [comment]
So it was written, so it shall be done. @jrastaban do you wanna edit this PR to just add the WP Solr plugin, without the images?

File: source/content/plugins.md [hunk 1]
```diff
@@ -34,30 +33,22 @@ The **Clear Cache** option will clear the cache for the entire site, but it does
...
 
 ## [Pantheon Advanced Page Cache](https://wordpress.org/plugins/pantheon-advanced-page-cache)
-![Page Cache plugin banner](../images/plugins/page_cache.png)
 Automatically clear related pages from Pantheon's Edge when you update content. Without this plugin, pages expire from cache after 10 minutes (600 seconds) by default. This plugin allows fresh content to be immediately served to anonymous visitors.
 
 ## [Pantheon HUD](https://wordpress.org/plugins/pantheon-hud)
-![HUD plugin banner](../images/plugins/hud.png)
 Provides situational awareness within the WordPress Dashboard when working on the Pantheon platform. It's helpful to have a reminder of which environment you're in, as well as quick access to links to get back to Pantheon's Dashboard, or to interface with your WordPress installation via the command line:
 
-![Pantheon HUD](../images/pantheon-hud.png)
```
Comment (alexfornuto) [inline source/content/plugins.md:41]: This wasn't a banner, but a pre-existing image showing the plugin in use.

File: source/content/plugins.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 title: Pantheon Plugins
 description: Details on specific WordPress plugins developed and maintained for the Pantheon Website Management Platform workflow.
-tags: [siteintegrations, infrastructure, cacheedge]
-categories: [performance, integrate,platform]
+tags: [WordPress, plugins, must-use, cache, SAML, Redis, Solr]
+categories: [integrate]
```
Comment (alexfornuto) [inline source/content/plugins.md:5]: Should we also add a review date?

**Comment by alexfornuto** [comment]
The PR still includes all the now-unused banner files.

**Pull request merged**